.github/ISSUE_TEMPLATE/new-trait-suggestion.md

```diff
@@ -0,0 +1,87 @@
+---
+name: New trait suggestion
+description: Suggest a new trait to add to the AusTraits Plant Dictionary (APD)
```

Comment:
This is just a new trait. What do we do if we want to modify existing trait? Maybe need a seperate template for that. Let's merge this in then can add others. I imagine a common request maybe to add a link to another external resource, which would ideally come as a pull request